exercise_02/getTotalWordsFromString.js

```diff
@@ -3,3 +3,14 @@
  * input: string (sentence)
  * output: number
  */
+
+/**
+ * Get the total number of words contained in one string.
+ * @param {string} sentence
+ * @returns number
+ */
+const getTotalWordsFromString = function(sentence) {
```

Comment:
This implementation is not compliant with requirements.

```js
getTotalWordsFromString("hello world") //ouput: 11 👉 EXPECTED: 2
```
Did you test it?

- [ ] Fix logic